### PR TITLE
Change domain name to `io.spine`.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: gcs-plugin/plugin/src/test/java/io/spine/tools/gcs/CleanGcsTaskShould.java [hunk 1]
```diff
@@ -45,9 +45,7 @@
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.spy;
-import static org.spine3.tools.gcs.CleanGcsTask.FOLDER_DELIMITER;
-import static org.spine3.tools.gcs.Given.createCleanGcsTask;
-import static org.spine3.tools.gcs.Given.newProject;
```
Comment (dmdashenkov) [inline gcs-plugin/plugin/src/test/java/io/spine/tools/gcs/CleanGcsTaskShould.java]: Why can't we continue using the `import static`?

File: spine-model-compiler/plugin/src/main/java/io/spine/gradle/compiler/lookup/enrichments/EnrichmentLookupPlugin.java [hunk 2]
```diff
@@ -120,7 +120,7 @@ private static void findEnrichmentsAndWriteProps(
         final DescriptorSetUtil.IsNotGoogleProto protoFilter =
                 new DescriptorSetUtil.IsNotGoogleProto();
         final Collection<FileDescriptorProto> files = getProtoFileDescriptors(descriptorSetPath,
-                                                                              protoFilter);
+                                                                                                protoFilter);
```
Comment (dmdashenkov) [inline spine-model-compiler/plugin/src/main/java/io/spine/gradle/compiler/lookup/enrichments/EnrichmentLookupPlugin.java]: Please undo the change.

**Comment by dmdashenkov** [comment]
@alexander-aleksandrov, please see the __codacy__ output. There is only one issue. It points onto an unused import which should be fixed.

File: spine-model-compiler/plugin/src/main/java/io/spine/gradle/compiler/lookup/enrichments/EnrichmentLookupPlugin.java (hunk 2 above)
Comment (alexander-aleksandrov) [inline spine-model-compiler/plugin/src/main/java/io/spine/gradle/compiler/lookup/enrichments/EnrichmentLookupPlugin.java]: Done

File: gcs-plugin/plugin/src/test/java/io/spine/tools/gcs/CleanGcsTaskShould.java (hunk 1 above)
Comment (alexander-aleksandrov) [inline gcs-plugin/plugin/src/test/java/io/spine/tools/gcs/CleanGcsTaskShould.java]: Fixed

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL
